packages/template/vite/tmpl/vite.main.config.mjs

```diff
@@ -1,4 +1,9 @@
 import { defineConfig } from 'vite';
 
 // https://vitejs.dev/config
-export default defineConfig({});
+export default defineConfig({
+  resolve: {
+    browserField: false,
```

Comment:
// Some libs that can run in both Web and Node.js envs, such as `axios`, we need to tell Vite to build them in Node.js env.